server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts

```diff
@@ -591,19 +607,24 @@ export class SummaryWriter implements ISummaryWriter {
                     $lt: lt,
                 },
             };
-            const logTail = await this.opStorage.find(query, { "operation.sequenceNumber": 1 });
-
-            // If the db is not updated with all logs yet, get them from checkpoint messages.
-            if (logTail.length !== (lt - gt - 1)) {
-                const nextSeq = logTail.length === 0 ? gt : logTail[logTail.length - 1].operation.sequenceNumber + 1;
-                for (const message of pending) {
-                    if (message.operation.sequenceNumber >= nextSeq && message.operation.sequenceNumber < lt) {
-                        logTail.push(message);
-                    }
+
+            const logTailOpMessage = await this.opStorage.find(query, { "operation.sequenceNumber": 1 });
```

Comment:
>opStorage [](http://example.com/codeflow?start=48&length=9)

I assume this is local DB?

Reply:
That is correct. This code path will not be exercised when we enable the getDeltasViaAlfred flag.